content/java/concepts/errors/terms/stringindexoutofboundsexception/stringindexoutofboundsexception.md

```diff
@@ -0,0 +1,40 @@
+---
+Title: 'StringIndexOutOfBoundsException'
+Description: 'Occurs when a String method tries to use an index that is either negative or greater than the size of the string.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Errors'
+  - 'Error Handling'
+  - 'Exceptions'
+  - 'Debugging'
+  - 'Strings'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The `StringIndexOutOfBoundsException` is an [exception](https://www.codecademy.com/resources/docs/general/error) that occurs when a [`String`](https://www.codecademy.com/resources/docs/java/strings) method tries to use an index that is either negative or greater than the size of the `String`.
+
+## Example
+
+The following example creates a `StringIndexOutOfBoundsException`:
+
+```java
+public class ErrorExample {
+  public static void main(String[] args) {
+    String greeting = "Hello World";
+    System.out.println(greeting.charAt(-1));
+  }
+}
+```
+
+The ouput is something like this:
+
```

Comment:
You can add a `<!-- markdownlint-disable -->` flag above this line to get the "lint" check to pass.

Reply:
@Dusch4593, I just eliminated the tabs.

Reply:
Ok